Sources/Spark/SparkScanModule.swift

```diff
@@ -152,7 +159,10 @@ open class SparkScanModule: NSObject, FrameworkModule {
                 result.reject(error: error)
                 return
             }
-            result.success(result: nil)
+
+            if !result.isResolved() {
+                result.success(result: nil)
+            }
```

Comment:
Explanation in https://github.com/Scandit/scandit-react-native-datacapture-core/pull/17